tests/plugins/inlinetoolbar/manual/focus.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, bug, inlinetoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,inlinetoolbar,sourcearea,list,link,image
+
+1. Click on image in the editor to open inline toolbar.
```

Comment:
Actually sometimes you need to click twice, this is the case for me on IE11 (but not only).